docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md

```diff
@@ -95,6 +97,8 @@ cleos push action eosio.nft.ft setprchsreq.a '[
     },
     "sale_shares": [],
     "maximum_uos_payment": "2 UOS",
+    start: "2023-09-18T13:21:10.724",
+    end: "2023-11-18T13:21:10.724",
```

Comment:
Must put the `start` and `end` into quotes.

While at it, please update the `maximum_uos_payment` and `price` to use 8 digit prevision: "2.00000000 UOS"